layouts/partials/head-meta.html

```diff
@@ -16,6 +16,6 @@
 <meta name="yandex-verification" content="{{ . }}" />
 {{ end }}
 
-{{- .Hugo.Generator -}}
```

Comment:
Are you sure about this change? 

Seems you are going from a context value to a function? But shouldn't a function be :
```
{{- hugo .Generator -}}
```

Don't know hugo, just from what I know of html/template.

Reply:
Yeah it was deprecated in v0.55. This is the suggested update approach: https://discourse.gohugo.io/t/pages-hugo-is-deprecated-as-of-0-55-0/17991